scripts/e2e-test.sh

```diff
@@ -168,6 +178,7 @@ echo "    e2e_image_tag=$e2e_image_tag"
 echo "    e2e_docker_registry=$e2e_docker_registry"
 echo "    e2e_reports_dir=$e2e_reports_dir"
 echo "    e2e_uninstall_cleanup=$e2e_uninstall_cleanup"
+echo "    test_plan=$test_plan_arg"
```

Comment:
The unofficial convention is environment variables used to control go code behaviour, are prefixed with e2e_